tools/misc/xenpm.c

```diff
@@ -48,11 +48,11 @@ void show_help(void)
             " get-cpufreq-average   [cpuid]       average cpu frequency since last invocation\n"
             "                                     for CPU <cpuid> or all\n"
             " get-cpufreq-para      [cpuid]       list cpu freq parameter of CPU <cpuid> or all\n"
-            " set-scaling-maxfreq   [cpuid] <HZ>  set max cpu frequency <HZ> on CPU <cpuid>\n"
+            " set-scaling-maxfreq   [cpuid] <kHZ> set max cpu frequency <kHZ> on CPU <cpuid>\n"
```

Comment:
I guess it would be useful to provide some explanation in this commit message why you have changed HZ to kHZ.  "Clarify xenpm usage" does not seem to be clear enough.

Reply:
because of mistake. I will provide.